test/http2_test/http2_test_server.py

```diff
@@ -0,0 +1,57 @@
+"""
+  HTTP2 Test Server. Highly experimental work in progress.
+"""
+import argparse
+import logging
+
+from twisted.internet.protocol import Factory
+from twisted.internet import endpoints, reactor
```

Comment:
These should be reordered (likewise with the local imports in this and the remaining files)

Possibly add a blank line separating these and the local imports